metron-interface/metron-rest/src/test/java/org/apache/metron/rest/controller/StormControllerIntegrationTest.java

```diff
@@ -182,6 +185,16 @@ public void test() throws Exception {
             .andExpect(jsonPath("$.status").value("SUCCESS"))
             .andExpect(jsonPath("$.message").value(TopologyStatusCode.STARTED.name()));
 
+    this.mockMvc.perform(get(stormUrl + "/supervisors").with(httpBasic(user,password)))
+            .andExpect(status().isOk())
+            .andExpect(content().contentType(MediaType.parseMediaType("application/json;charset=UTF-8")))
+            .andExpect(jsonPath("$.supervisors[0]").exists())
+            .andExpect(jsonPath("$.supervisors[0].id").exists())
+            .andExpect(jsonPath("$.supervisors[0].host").exists())
+            .andExpect(jsonPath("$.supervisors[0].upTime").exists())
```

Comment:
Which API spec are you referring to?  This [page](http://storm.apache.org/releases/1.0.3/STORM-UI-REST-API.html) has it as "uptime".